pkg/apis/cstor/v1/cstorbackuprestorebuilder.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+// IsFailed returns true if backup failed
+func (backup *CStorBackup) IsFailed() bool {
+	return backup.Status == BKPCStorStatusFailed
+}
+
+// IsSucceeded returns true if backup completed successfully
+func (backup *CStorBackup) IsSucceeded() bool {
+	return backup.Status == BKPCStorStatusDone
+}
+
+// IsPending return true if the backup is in pending state
+func (backup *CStorBackup) IsPending() bool {
+	return backup.Status == BKPCStorStatusPending
+}
+
+// IsInProgress return true if the backup is in progress state
+func (backup *CStorBackup) IsInProgress() bool {
+	return backup.Status == BKPCStorStatusInProgress
+}
+
+// IsInitilized return true if the backup is in init state
+func (backup *CStorBackup) IsInitilized() bool {
```

Comment:
n_p: Spelling?
Infact we can make it `IsInInit` IMO

Reply:
I am nit picking. Also -- while you are fixing. Can you please check the comments too.
e.g. `IsInProgress returns...` 
Similarly at other places..

Reply:
> Infact we can make it IsInInit IMO

IsInit (or) IsInInit? What is meant by IsInInit?

Reply:
I got it and I will update to IsInInit